app/src/main/java/org/sopt/official/feature/attendance/NewAttendanceViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package org.sopt.official.feature.attendance
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+import org.sopt.official.domain.repository.attendance.AttendanceRepository
+import org.sopt.official.feature.attendance.model.AttendanceUiState
+import javax.inject.Inject
+
+@HiltViewModel
+class NewAttendanceViewModel @Inject constructor(
+    private val attendanceRepository: AttendanceRepository
+) : ViewModel() {
+
+    private val _uiState: MutableStateFlow<AttendanceUiState> = MutableStateFlow(AttendanceUiState.Loading)
+    val uiState: StateFlow<AttendanceUiState> = _uiState
+
+    private var fakeTitle: String = ""
+
+    fun updateUiState() {
+        viewModelScope.launch {
+            _uiState.emit(AttendanceUiState.Success(fakeTitle))
+        }
+    }
+
+    fun onClickFake() {
+
+    }
```

Comment:
이 친구는? 만약에 아무 행동도 정의안할 함수면

```suggestion
    fun onClickFake() = Unit
```

혹은

```suggestion
    fun onClickFake() {
        // TODO by Junseo 구현해아할 함수
    }
```
정도로 적을 수 있을 것 같습니다.

Reply:
요 부분은 임의로 만들어둔거라 삭제하도록 하겠습니다 !